x-pack/plugins/apm/public/components/shared/charts/TransactionCharts/index.tsx

```diff
@@ -81,27 +85,30 @@ export function getMaxY(responseTimeSeries: TimeSeries[]) {
   return Math.max(...numbers, 0);
 }
 
-export class TransactionCharts extends Component<TransactionChartProps> {
-  public getTPMFormatter = (t: number) => {
-    const { urlParams } = this.props;
+export function TransactionCharts({
+  charts,
+  location,
+  urlParams,
+}: TransactionChartProps) {
+  const getTPMFormatter = (t: number) => {
```

Comment:
Great to see this being converted from a class component to a functional component.